CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to
 
 ## [Unreleased]
 
+## [1.0.0] - 2022-05-13
+
+- BREAKING: rename property `target` -> `targets` for BountySubmissionEntity
```

Comment:
I'm not sure whether anyone is using this property, but it will break existing queries if so.

Reply:
I was able to confirm that there are no saved queries involving bugcrowd specifying this property

Reply:
nor were any made in past week